compose-dev-frontend.yml

```diff
@@ -42,5 +42,6 @@ services:
             - EDIT_TIME=5m # edit window
             - AUTH_ANON=true
             - AUTH_EMAIL_ENABLE=true
+            - SIMPLE_VIEW=true
```

Comment:
this may confuse others. For such custom tests, I usually make a private compose file. Some already listed in .gitignore:
```
compose-private-backend.yml
compose-private-frontend.yml
compose-private.yml
```

Reply:
Thanks, I going to do it.